src/app/auth/containers/auth/auth.component.ts

```diff
@@ -7,9 +9,37 @@ import { AuthService } from '../../services/auth.service';
   styleUrls: ['./auth.component.scss']
 })
 export class AuthComponent implements OnInit {
-
-  constructor() { }
+  constructor(private authService: AuthService) { }
+  signupVisible = false;
 
   ngOnInit() { }
 
+  toggle() {
+    this.signupVisible = !this.signupVisible;
+  }
+
+  login(userCredentials: UserCredentials) {
+    this.authService.login(userCredentials).subscribe(user => {
```

Comment:
we would need to unsubscribe also, we could use rxjs take operator to only take the first value back from that API call